browser/ntp_background/ntp_p3a_helper_impl.cc

```diff
@@ -86,28 +110,58 @@ NTPP3AHelperImpl::NTPP3AHelperImpl(PrefService* local_state,
   rotation_subscription_ =
       p3a_service->RegisterRotationCallback(base::BindRepeating(
           &NTPP3AHelperImpl::OnP3ARotation, base::Unretained(this)));
+  auto* ntp_background_images_service =
+      g_brave_browser_process->ntp_background_images_service();
+  if (ntp_background_images_service) {
+    const auto* sr_data =
+        ntp_background_images_service->GetBrandedImagesData(true);
+    const auto* si_data =
+        ntp_background_images_service->GetBrandedImagesData(false);
+    if (sr_data) {
```

Comment:
this can be rewritten as:

```
if (const auto* sr_data =
        ntp_background_images_service->GetBrandedImagesData(true)) {
  CheckLoadedCampaigns(*sr_data);
}
```